src/pcp/types.rs

```diff
@@ -426,7 +426,7 @@ impl<F: FieldElement, S> Clone for CountVec<F, S> {
 
 impl<F, S> Type for CountVec<F, S>
 where
-    F: FieldElement + Send + Sync,
```

Comment:
Evidently not: if I run `cargo test --all-features` then `pcp::gadgets::tests::test_parallel_sum_multithreaded` compiles and passes. I can put this trait bound back, because `FieldElement` will almost certainly always be `Send + Sync` and it would be convenient for crate `prio` to do this because it spares clients who want to use this in a multithreaded context from writing out some crazy trait bounds. [Here](https://github.com/abetterinternet/ppm-prototype/blob/a0ed255336762ac1ef1ea08b459cb725996851d5/src/leader.rs#L462)'s an example: I have to put `Send + Sync` requirements on seven different types to be able to generically clone an aggregator across threads.

At that point, we might put `Send + Sync` trait bounds in a great many places as a convenience to clients of the crate but I don't want to do that in this PR.